CMakeLists.txt

```diff
@@ -361,4 +361,8 @@ install( FILES LICENSE README.md NEWS
     DESTINATION ${CMAKE_INSTALL_DOCDIR}
     )
 
+add_custom_target( install-nodoc
+  COMMAND make NEST_INSTALL_NODOC=true install
```

Comment:
Can I still use `-j N` with this?

Reply:
The short answer is yes.

The slightly longer answer is that my changes do not alter the behavior of your call to `make` for any target but `install`. And for that `-j N` did not have an effect anyway as the installation is always performed serially in order to avoid problems with concurrent writes.